src/test/java/seedu/address/testutil/TypicalTags.java

```diff
@@ -0,0 +1,22 @@
+package seedu.address.testutil;
+
+import java.util.HashSet;
+import java.util.Set;
+
+import seedu.address.model.tag.Tag;
+
+/**
+ * A utility class containing a set of {@code Tag} objects to be used in tests
+ */
+public class TypicalTags {
+    public static final Tag FRIENDS = new Tag("friends");
+
+    public static final Tag OWES_MONEY = new Tag("owesMoney");
```

Comment:
Would it be better to have the variables declared in CommandTestUtil instead? We may also change the variable name to agree with those already there, e.g. VALID_TAG_FRIENDS.